openfisca_us/variables/gov/states/in/tax/income/exemptions/in_aged_low_agi_exemptions.py

```diff
@@ -0,0 +1,28 @@
+from openfisca_us.model_api import *
+
+
+class in_aged_low_agi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN base exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states["in"].tax.income.exemptions
+        filing_status = tax_unit("filing_status", period)
+        threshold = p.aged_low_agi.threshold[filing_status]
+        aged_low_agi_exemption = p.aged_low_agi.amount
+        federal_agi = tax_unit("adjusted_gross_income", period)
+        age_threshold = parameters(
+            period
+        ).gov.irs.deductions.standard.aged_or_blind.age_threshold
+        aged_head = (tax_unit("age_head", period) >= age_threshold) * 1
```

Comment:
Similar to above, could you make `aged_head` and `aged_spouse` variables in the `irs` folder, since we're starting to duplicate code from there?